scss/civicrm/event/_detail.scss

```diff
@@ -0,0 +1,243 @@
+#crm-container {
+  input[type=radio]:not(.old-radiocheckbox-style):checked+label[for]::after {
+    top: 7px;
+  }
+
+  button {
+    background: $brand-primary;
+    border: 0;
+    border-radius: $border-radius-base;
+    color: $crm-pure-white-color;
+    font-family: $font-family-base;
+    height: 36px;
+    line-height: 1;
+    padding: 11px 12px;
+    text-shadow: none;
+    text-transform: uppercase;
+    vertical-align: middle;
+
+    &:hover {
+      background: darken($brand-primary, $crm-darken-percentage);
+    }
+
+    &:active,
+    &:focus {
+      outline: none;
+    }
+
+    &.cancel {
+      background: $crm-white;
+      border: 1px solid $crm-copy;
+      color: $crm-cancel-color;
+
+      &:hover {
+        background: $crm-copy;
+        color: $crm-white;
+      }
+    }
+  }
+
+  .crm-actions-ribbon {
+    margin-bottom: 20px;
+  }
+
+  .crm-form-block.crm-form-block>.crm-submit-buttons,
+  .crm-form-block .form-item>.crm-submit-buttons {
```

Comment:
This section is stlyed globally and should not be re-styled here

Reply:
But we don't need this border anymore:
![no-border-need](https://user-images.githubusercontent.com/3272950/34294888-3e76d4ce-e713-11e7-9afb-aca66fa0eed9.png)

Reply:
@mukeshcompucorp where is it styled? Can we add an exception in that partial to remove the border in case the element it's inside tabs? Let's not scatter styles for the same element all over the place

Reply:
@AkA84 @boriszaydlin This was a part of NDI-15 https://github.com/civicrm/org.civicrm.shoreditch/pull/101 which is still pending to merge and review.

Reply:
Border still exists, so I will keep this code